<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CopyfieldsWidget</class>
 <widget class="QWidget" name="CopyfieldsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>28</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QComboBox" name="functionCombo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>Where</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Update</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="srcCombo"/>
   </item>
   <item>
    <widget class="QLabel" name="signLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="destCombo"/>
   </item>
   <item>
    <widget class="QToolButton" name="deleteButton">
     <property name="text">
      <string>Delete</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>.</normaloff>.</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>functionCombo</tabstop>
  <tabstop>srcCombo</tabstop>
  <tabstop>destCombo</tabstop>
  <tabstop>deleteButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>CopyfieldsWidget</receiver>
   <slot>deleteLater()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>14</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
